### DISCUSSION TITLE
How do i connect rainmeter for spotx

### DISCUSSION THREAD

**Discussion by Jaaaaabs** [SpotX-Official/SpotX#466] [Question]
so i got rainmeter but the player is not working is there a way to connect it??

**Comment by amd64fox** [comment]
￶￶￶Spotx does not break the player in the native Sootify client, if it worked for you before, then it should work together with Spotx.

**Comment by tegardewaa** [comment]
> so i got rainmeter but the player is not working is there a way to connect it??

Hi, I had the same problem as you but after I changed to old_theme everything worked normally again, you can also try it, I hope it helps.
